src/sparse/impl/KokkosSparse_spgemm_impl_speed.hpp

```diff
@@ -313,35 +315,46 @@ struct KokkosSPGEMM
           shmem_key_size = shmem_key_size + ((shmem_key_size - shmem_hash_size) * sizeof(nnz_lno_t)) / (sizeof (nnz_lno_t) * 2 + sizeof(scalar_t));
           shmem_key_size = (shmem_key_size >> 1) << 1;
 
+// This guard will help ensure behavior is consistent within Trilinos
+#ifdef KOKKOS_ENABLE_COMPLEX_ALIGN
```

Comment:
Where is this defined ? In Kokkos ?

Reply:
@srajama1 yes

Reply:
Ok, I wish there is a better alternative than using Kokkos macros, but this is fine.